package.json

```diff
@@ -75,8 +76,7 @@
     "react-dom": "^15.3.2",
     "react-hot-loader": "^1.3.0",
     "react-redux": "=4.4.3",
```

Comment:
We can now unlock this dependency (change it back to `^` instead of `=`), because react-router 3 fixes the conflict with an optimization done in react-redux 4.4.4.